package.json

```
@@ -41,6 +41,6 @@
   },
   "lint-staged": {
     "*.{ts,js}": ["biome lint --write", "biome format --write"],
-    "*.json": "biome format --write"
+    "*.json": "biome format --write --no-errors-on-unmatched"
```

An error was being thrown because no `.json` files are in the code base. Adding this flag will suppress the error.